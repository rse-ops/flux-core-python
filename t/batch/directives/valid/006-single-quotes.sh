#!/bin/sh
# flux: --setattr=user.data='{"option": "arg"}'
