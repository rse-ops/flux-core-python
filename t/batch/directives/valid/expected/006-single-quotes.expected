SETARGS(['--setattr=user.data={"option": "arg"}'])
